.github/workflows/_lints.yml

```
@@ -0,0 +1,77 @@
+name: Reusable workflow for lints
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+
+jobs:
+  clippy:
+    name: Clippy (MSRV)
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.target }}
+    steps:
+      - uses: actions/checkout@v3
```

I suggest that exact versions are used where possible such as

```suggestion
      - uses: actions/checkout@v3.5.2
```

and Dependanabot is configured to update versions as done [here](https://github.com/JarvisCraft/mc-server-pinger/blob/4d9248575e24a6c914e612ba3083d6f89ec2887d/.github/dependabot.yml#L25-L34) (see PR history in the repo to see how it looks).

It will allow better reproducibility while keeping up to date.

---

Yep, Dependabot config is coming next once I get the lints working.

---

Dependabot config has been added.